### PR TITLE
Tolerant variable reference finder

### PR DESCRIPTION
**Pull request by BladeMF**
Implementing https://github.com/phpactor/phpactor/issues/1164

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,6 @@
+/vendor
+/phpbench.json
+/composer.lock
+/.php_cs.cache
+/stubs
```
Comment (dantleech) [inline .gitignore:4]: `phpbench.json` shouldn't be ignored ..

Comment (BladeMF) [reply .gitignore:4]: Copied it from somewhere... Sorry about that.

**Comment by BladeMF** [comment]
Rebased on master

**Comment by BladeMF** [comment]
@dantleech Not sure what I am doing wrong here, if you could lend a hand. I did change the version of worse-reference-finders to `0.2.4` but doesn't seem to have did the trick.

**Comment by dantleech** [comment]
> Not sure what I am doing wrong here

not sure what you're trying to achieve. the next version of reference finders is `0.2.5` - but if you want your changes they will either need be merged or you need to include

```
"repositories": [
     {
          "type": "vcs",
          "url": "https://github.com/your/worse-reference-finder"
     }
]
```

in `composer.json` but probably easier to wait until it's merged.

**Comment by BladeMF** [comment]
My mistake. I got confused it was merged for some reason. I was under the impression that composer is just not picking up the latest changes.

**Pull request closed without merging**